src/python/grapl-common/BUILD

```diff
@@ -2,9 +2,6 @@ grapl_python_distribution(
     dependencies = [
```

Comment:
# This is the thing I lost an hour on - I'd have needed to add `/sqs` here and that's annoying and well hidden :(